pkgs/development/python-modules/pandoc/default.nix

```diff
@@ -0,0 +1,30 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, lib
+, plumbum
+, ply
+}:
+
+buildPythonPackage rec {
+  pname = "pandoc";
+  version = "2.3";
+
+  src = fetchFromGitHub {
+    owner = "boisgera";
+    repo = pname;
+    rev = version;
+    hash = "sha256-53LCxthxFGiUV5go268e/VOOtk/H5x1KazoRoYuu+Q0=";
+  };
+
+  propagatedBuildInputs = [
+    plumbum
+    ply
+  ];
+
```

Comment:
The upstream repo contains something that looks like a [unittest](https://github.com/boisgera/pandoc/blob/master/test.py). It's not possible to execute the tests, add `doCheck = false;` with a comment please.

Please add `pythonImportsCheck`.

Reply:
Done.